src/main/kotlin/org/rust/ide/refactoring/introduceVariable/impl.kt

```diff
@@ -134,7 +133,7 @@ private fun findAnchor(expr: PsiElement): PsiElement? {
     return anchor
 }
 
-private class RustInPlaceVariableIntroducer(
+class RustInPlaceVariableIntroducer(
```

Comment:
Maybe move the class to some common pace?

Reply:
Done, created `introduceUtils.kt` for such needs.